.ci/magician/cmd/check_cassettes.go

```diff
@@ -64,7 +64,7 @@ var checkCassettesCmd = &cobra.Command{
 
 		ctlr := source.NewController(env["GOPATH"], "modular-magician", githubToken, rnr)
 
-		vt, err := vcr.NewTester(env, rnr)
+		vt, err := vcr.NewTester(env, "ci-vcr-logs", "ci-vcr-cassettes", rnr)
```

Comment:
it looks like previously this was using `"vcr-check-cassettes"` - just wanted to double-check that this is intentionally changing?

Reply:
Unintentional change, reverting to the correct bucket now.